test/bench/bench.cpp

```diff
@@ -103,14 +103,27 @@ void benchmark_execute(benchmark::State& state, const ExecutionBenchmarkCase& be
             return state.SkipWithError("Incorrect result memory");
     }
 
-    for (auto _ : state)
+    for (auto _ : state)  // NOLINT(clang-analyzer-deadcode.DeadStores)
     {
         instance.memory = initial_memory;  // Restore initial memory.
         const auto result = fizzy::execute(instance, *func_idx, benchmark_case.func_args);
         benchmark::DoNotOptimize(result);
     }
 }
 
+template <typename Lambda>
+void register_benchmark(const std::string& name, Lambda&& fn)
+{
+#ifdef __clang_analyzer__
+    // Clang analyzer reports potential memory leak in benchmark::RegisterBenchmark().
```

Comment:
Not possible to disable it with `NOLINT` ?

Reply:
No, because the main error is in `benchmark.hpp`. There is option `HeaderFilterRegex` but does not seem to work. Maybe this option is only for clang-tidy checks – clang analyzer is different beast, just available in the same tool.